### ISSUE TITLE
Integrate with org-bullets-mode

### ISSUE THREAD

**Issue by eddsteel** [iqbalansari/emacs-emojify#25]
I use emoji for my org bullets, using [org-bullets](https://github.com/sabof/org-bullets) to customise them. When I look at the config with global emojify mode on I see that the emoji are correctly appearing. My config is:
```
(setq org-bullets-bullet-list '("🍣" "🐸" "🐳" "🐻" "◉" "○" "✸" "✿"))
```

When I look at an org-mode file I see the bullets appearing as boxes, i.e. emojify is not changing the characters to images. I assume this is caused by something about the way that org-bullets-mode works. It looks like it uses `compose-region` from `composite.el`. Is this the same thing as the `composition` property you mentioned in #16? I tried updating to latest in MELPA and setting `(emojify-set-emoji-style '(prettify-symbol ascii unicode github))` but it didn't help.

Let me know if you have any ideas! Thanks for this great project!

**Comment by iqbalansari** [comment]
Hi,

Glad you like the package and thanks for the bug report. It seems org-bullets uses the same mechanism as `prettify-symbol` how the implementation described in #16, is not general enough to handle `org-bullets`, I will need to think of (possibly) another strategy for `org-bullets`. Let me try again

Thanks

**Comment by iqbalansari** [comment]
Hi @eddsteel,

I have just pushed a general method for handling composed text on master branch. From my initial testing this seems to work well with `org-bullets` mode, can you give this a try after MELPA updates (can take some time) and report back.

Thanks

**Comment by eddsteel** [comment]
Looks great! Thanks so much.

**Comment by iqbalansari** [comment]
Great, I am closing this issue.

Thanks :smile:

**Comment by frondeus** [comment]
Hi, I can't properly setup emojify with org-bullets.
![emacs-emojify](https://user-images.githubusercontent.com/1165825/37103151-d7e2fd9e-2229-11e8-93fe-fc127f12b327.png)

As you can see, I used same config - copy-pasted it from OP example.
It does not matter if I use :github_style: or unicode_style.

When I use :github_style: it prints only ":".
When I use unicode_style: it prints rectangle.

I'm using GNU Emacs 25.3.1 (x86_64-w64-mingw32) of 2017-09-12 from Chocolatey on Windows 10.
I downloaded package from MELPA.

**Comment by frondeus** [comment]
It seems that problem with bullets has been resolved.
I used
```elisp
(require 'font-lock)
(require 'font-lock+)
```
before loading emojify.

Org-ellipsis is another issue.

**Comment by zhanghanghitomi** [comment]
Hi, there, 

I still met with the same problems. 

Currently, my setting is as 
idotspacemacs-configuration-layers
   '(
     better-defaults
     (c-c++ :variables c-c++-default-mode-for-headers 'c++-mode
            :variables c-c++-enable-clang-support t)
     markdown
     python
     octave
     ;; ----------------------------------------------------------------
     ;; Example of useful layers you may want to use right away.
     ;; Uncomment some layer names and press <SPC f e R> (Vim style) or
     ;; <M-m f e R> (Emacs style) to install them.
     ;; ----------------------------------------------------------------
     ivy
     (auto-completion :variables
                      auto-completion-return-key-behavior 'complete
                      auto-completion-tab-key-behavior 'cycle
                      auto-completion-complete-with-key-sequence nil
                      auto-completion-complete-with-key-sequence-delay 0.1
                      auto-completion-private-snippets-directory nil)
     bibtex
     ;; better-defaults
     emacs-lisp
     emoji
     git
     (latex :variables latex-build-command "LaTeX"
            :variables latex-enable-auto-fill t
            :variables latex-enable-folding t)
     markdown
     org
     journal
     ;; (shell :variables
     ;;        shell-default-height 30
     ;;        shell-default-position 'bottom)
     spell-checking
     syntax-checking 

     (osx :variables osx-dictionary-dictionary-choice "Simplified Chinese - English"
          osx-command-as 'super)

     ;; version-control
     ycmd
     ) 

(defun dotspacemacs/user-config ()
(setq org-bullets-bullet-list '(":whale:" ":tropical_fish:" ":dolphin:" ":fish:"))
)

And these emoji files can show correctly. 

But when I open the org file, the symbols becomes ":". 

I wonder whether the author could give some hint on how to solve this. 
If possible, could you share you spacemacs init file so I can check the difference my self. 

PS: based on my understanding, this problem should be solved with the new emojify package, 
which is contained in the emoji layer. 

However, I used the newest version but it seems does not work.

**Comment by iqbalansari** [comment]
Hi @zhanghanghitomi,

Looking at the [emoji layer](https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Bfun/emoji/packages.el) in Spacemacs it does not seem that it uses this (`emojify`) package for displaying emojis. You will have to install the package separately. I am not really familiar with Spacemacs so I am afraid I am not sure of the correct way of integrating `emojify` with it
